dokka-runners/dokka-gradle-plugin/src/testFunctional/kotlin/DokkaGeneratorLoggingTest.kt

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+package org.jetbrains.dokka.gradle
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.collections.shouldContainAll
+import io.kotest.matchers.string.shouldContain
+import io.kotest.matchers.string.shouldNotContain
+import org.jetbrains.dokka.gradle.internal.DokkaConstants.DOKKA_VERSION
+import org.jetbrains.dokka.gradle.utils.*
+
+class DokkaGeneratorLoggingTest : FunSpec({
+
+    context("DokkaGenerator logging:") {
+        val project = createProject()
+
+        test("at lifecycle log level expect only error and warn logs") {
+
+            project.runner
+                .addArguments(
+                    ":dokkaGenerateModuleHtml",
+                    "--rerun",
+                )
+                .build {
+                    output.invariantNewlines() shouldContain """
+                        > Task :dokkaGenerateModuleHtml
+                        e: [:dokkaGenerateModuleHtml] test error message
```

Comment:
TBH, I don't think that duplicating task name will be useful, but I'm not against having it :)

Reply:
I put the task name in because it can be confusing if there are multiple generations at the same time, or if running with `--quiet`. It's already a bit confusing when aggregating, because sometimes the same error is shown for both module and publication generation. But we can revisit the message if necessary.